contrib/core-contract-tests/tests/clar/clar-flow.test.ts

```diff
@@ -0,0 +1,138 @@
+import { ParsedTransactionResult, tx } from "@hirosystems/clarinet-sdk";
```

Comment:
Hey, do you think you can add some method-level comments for each of these functions in this file and the `clar.test.ts` file?  I'm having a hard time following how these functions fit together.  Thanks!

Reply:
This parser evolved over time during our work on sBTC Mini. We just discussed in the Clarity WG to clean it up and perhaps introduce a more robust parser. The current files have served our needs and were recently edited to support `clarinet-sdk`.

Reply:
Comments have been added